components/atom/button/src/index.js

```diff
@@ -5,7 +5,22 @@ import cx from 'classnames'
 import Button from './Button'
 
 const CLASS = 'sui-AtomButton'
-const TYPES = ['primary', 'accent', 'secondary', 'tertiary']
+const TYPES = ['solid', 'outline', 'flat']
```

Comment:
Where is the definition of all of this? I checked https://github.com/SUI-Components/UX-Definitions/blob/master/Atom/Buttons.md and types and colors are named different 🤷‍♂

Reply:
I just figured out that now our code > their definition